build-system/pr-check/build-targets.js

```diff
@@ -207,6 +208,9 @@ const targetMatchers = {
       file.startsWith('build-system/test-configs')
     );
   },
+  [Targets.JSON_FILES]: (file) => {
```

Comment:
In addition to the `.json` files, also include `build-system/tasks/check-json-schemas.js` in the matcher function so that the check is triggered during CI if its implementation were to change.